modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-aws-amis.adoc
+
+[id="rosa-classic-aws-rhcos-ami_{context}"]
+= ROSA AMIs for {product-title}
+
+Red Hat provides RHCOS AMIs that are valid for the supported AWS regions and instance types that you can manually specify for your {product-title} nodes.
+
+.{product-title} AMIs
+
+[cols="2a,2a,2a",options="header"]
+|===
+
+|AWS zone
+|AWS AMI
+|Description
+
+.19+|*us-east-1*
```

Comment:
I know this is probably a lot of work, but I think it might be more helpful to split this table up by version (that allows customers to click on the version their cluster was installed in which takes them straight to the AMI list) so we can easily add and remove new sections....

Reply:
So, you want tables separated by version, and then list the AMIs for each region in these tables? Is that correct?

Reply:
Yep, exactly!

Reply:
I'd +1 this. Maybe a future enhancement could be some kind of "interactive" functionality where a customer can select a version and/or a region and it would just filter the image IDs for them?

This could also potentially be an enhancement to the OCM API so they could just query the endpoint (and it would be one less docs change to consistently maintain as versions grow?)